README.md

```diff
@@ -39,18 +38,52 @@ Report issues, ask questions, and provide suggestions using:
 
 You may reach out to project maintainers privately at onedal.maintainers@intel.com
 
-# Installation
-daal4py can be installed from conda-forge (recommended):
+# 🛠 Installation
+daal4py is available at the [Python Package Index], on Anaconda Cloud in [Conda-Forge channel](https://anaconda.org/conda-forge/daal4py) and in [Intel channel](https://anaconda.org/intel/daal4py) .
 ```bash
+# PyPi
+pip install daal4py
+```
+```bash
+# Anaconda Cloud from Conda-Forge channel
 conda install daal4py -c conda-forge
 ```
-or from Intel channel:
 ```bash
+# Anaconda Cloud from Intel channel
```

Comment:
I think we need to specify where conda-forge and intel channels are recommended.

Reply:
Done